ixwebsocket/IXWebSocketTransport.cpp

```diff
@@ -565,6 +591,13 @@ namespace ix
             _rxbuf.erase(_rxbuf.begin(),
                          _rxbuf.begin() + ws.header_size + (size_t) ws.N);
         }
+
+        // if an abnormal closure was raised in poll, and nothing else triggered a CLOSED state in
+        // the received and processed data, then close uising abnormal close code and message
+        if (_readyState != CLOSED && _readyState != CLOSING && pollPostTreatment == CHECK_OR_RAISE_ABNORMAL_CLOSE_AFTER_DISPATCH)
```

Comment:
please break down long conditional on 2 lines.

Reply:
_readyState != CLOSED && _readyState != CLOSING => can we use isConnected() instead ?

Reply:
Ok and maybe, I have to check